docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -0,0 +1,228 @@
+    ```
+
+=== "PostgreSQL"
+
+    ``` bash
+    psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/commerce/ibexa-4.5.latest-to-4.6.0.sql
+    ```
+
+## Run data migration
+
+### Dashboard [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+If you are using [[= product_name_exp =]] or [[= product_name_com =]],
+you must run data migration required by the dashboard and other features to finish the upgrade process:
+
+```bash
+php bin/console ibexa:migrations:import vendor/ibexa/dashboard/src/bundle/Resources/migrations/structure.yaml --name=2023_09_23_14_15_dashboard_structure.yaml
+php bin/console ibexa:migrations:import vendor/ibexa/dashboard/src/bundle/Resources/migrations/permissions.yaml --name=2023_10_10_16_14_dashboard_permissions.yaml
+php bin/console ibexa:migrations:migrate --file=2023_09_23_14_15_dashboard_structure.yaml --file=2023_10_10_16_14_dashboard_permissions.yaml
+```
+
+## Revisit mandatory configuration
+
+### Dashboard [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+Define "Dashboards" location as contextual tree root:
+
+```yaml
+ibexa:
+    system:
+        # ...
+        admin_group:
+            content_tree_module:
+                contextual_tree_root_location_ids:
+                    #...
+                    - 67 # Dashboards (clean installation)
+```
+
+### User profile
+
+Ibexa DXP v4.6 introduced user profile for Backoffice users, allowing to ...
+
+This feature is optional, and you might disable it by setting `enabled` flag to `false` in `ibexa.system.<scope>.user_profile` configuration:
+
+```yaml
+# /config/packages/ibexa_admin_ui.yaml
+ibexa:
+    system:
+        # ...
+        admin_group:
+            user_profile:
+                enabled: false
+```
+
+In order to enable user profile, you need to se specify content type identifiers which are representing editor user and field groups that will be rendered in user profile summary:
+
+```yaml
+# /config/packages/ibexa_admin_ui.yaml
+ibexa:
+    system:
+        # ...
+        admin_group:
+            user_profile:
+                enabled: true
+                content_types: ['editor']
+                field_groups: ['about', 'contact']
+```
+
+You can use your own content type representing backoffice user or use default one provided by Ibexa DXP:
+
+```bash
+php bin/console ibexa:migrations:import vendor/ibexa/installer/src/bundle/Resources/install/migrations/2023_12_07_20_23_editor_content_type.yaml --name=2023_12_07_20_23_editor_content_type.yaml
+php bin/console ibexa:migrations:import vendor/ibexa/installer/src/bundle/Resources/install/migrations/2024_01_09_22_23_editor_permissions.yaml --name=2024_01_09_22_23_editor_permissions.yaml
+php bin/console ibexa:migrations:migrate --file=2023_12_07_20_23_editor_content_type.yaml --file=2024_01_09_22_23_editor_permissions.yaml
+```
+
+More information ...
```

Comment:
```suggestion
More information ...
```
Do you remember what you wanted to say here?

Reply:
@adamwojs Do you remember what kind of extended information you were thinking of?

Reply:
Removed